docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -159,6 +162,29 @@ Heterogeneous plugin was tested with the Intel&reg; GNA as a primary device and
 
 > **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
+## Support for Quality of Service (QoS) Mechanism on Windows\*
+
```

Comment:
Again, I would not call it "QoS".  Rather, I would title the section "Recovering from interruption by high priority Windows audio processes".

Reply:
Fixed